packages/components/src/date-time/test/date.js

```diff
@@ -1,97 +1,111 @@
 /**
  * External dependencies
  */
-import { shallow } from 'enzyme';
 import moment from 'moment';
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
FYI we now have [`user-event`](https://testing-library.com/docs/user-event/intro) in this repo, so it's probably better for us to use `userEvent` instead of `fireEvent` by default.